src/code42cli/cmds/devices.py

```diff
@@ -297,6 +297,10 @@ def handle_row(guid):
         return device_dataframe
 
 
+def _filter_dataframe_by_date_column(df, column, start, end):
```

Comment:
This method does not filter anything

Reply:
yeah, that was an inadvertent commit from another in-progress branch.